package.json

```diff
@@ -67,12 +64,11 @@
       "no-template-curly-in-string": 0,
       "prefer-const": 0,
       "no-unused-vars": 0,
-      "flowtype/define-flow-type": 2,
       "import/no-duplicates": 0,
       "prettier/prettier": [
         "error",
         {
-          "parser": "flow"
+          "parser": "typescript-eslint"
```

Comment:
Interesting - I would have assumed that javascript/typescript/babel-ts parsers would work here. I understand the difference between them but I don't immediately see why there is a need for Prettier's typescript-eslint to exist in the first place. Is this just about making sure what both use the same baseline of features? So if something can be parsed using typescrip-teslint then it can also be parsed using Prettier? I suspect that otherwise we could end up in a situation with a single file being parseable by one without being parseable by the other one.

Reply:
Yeah, they're pretty equivalent. For now it's all vanilla JavaScript, which TypeScript supports. I'd think long-term it'd be easier to just have one parser for all of them.